README.md

```diff
@@ -33,6 +33,8 @@ protocol and port may be omited, `http` and `6060` are used by default
 
 * `DOCKER_PASSWORD` - Docker registry account password
 
+* `KLAR_INSECURE` - Allow Klar to access registries with bad SSL certificates. Default is `false`.
```

Comment:
`DOCKER_INSECURE` is better? In addition, if want support insecure Docker registry, we should remind the users to config the Clair server with 'insecure-tls=true'.